lms/djangoapps/certificates/migrations/0001_initial.py

```diff
@@ -116,7 +115,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('course_id', CourseKeyField(default=None, max_length=255, blank=True)),
                 ('whitelist', models.BooleanField(default=0)),
-                ('created', django_extensions.db.fields.CreationDateTimeField(auto_now_add=True, verbose_name='created')),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, verbose_name='created', editable=False)),
```

Comment:
Looks like the default is already `django.utils.timezone.now`, does this need to be explicit if "Created" is already in the class name?

Reply:
oh, is this an automatically generated file?

Reply:
Correct, this file is generated by makemigrations command.

Reply:
To be specific, the file was automatically generated back when we were using django-extensions (which generated the line being replaced here).  I'm replacing that line with the output that has been generated in other migration files where we were already using `AutoCreatedField`.  Rewriting history like this is ok in this case because both lines yield the exact same database schema.  There will be some subtle differences in exactly when the timestamp gets created and what happens if you try to change the timestamp after the model instance has already been created and saved, but it's extremely unlikely that we have any code that really cares about those details.

I could have made the behavior match exactly what it was doing before, but then it would be inconsistent with the behavior of all our other `created` and `modified` fields, which use this default behavior from `django-model-utils`.